### PR TITLE
interfaces/builtin/dsp: update proc files for ambarella flavor

### PR DESCRIPTION
**Pull request by woodrow-shen**
There are two commits that just update ambarella flavor to add proc files to support their new features, and meanwhile rectify some permissions of proc files.

### CODE REVIEW

**Review by mvo5 — APPROVED** [review]
This looks fine to me

**Review by mardy — APPROVED** [review]
LGTM!

File: interfaces/builtin/dsp.go [hunk 1]
```diff
@@ -61,16 +61,17 @@ const ambarellaDspConnectedPlugApparmor = `
...
-/proc/ambarella/clock rw,
+/proc/ambarella/iav r,
+/proc/ambarella/ambnl/** rw,
+/proc/ambarella/udc r,
+/proc/ambarella/clock r,
 /proc/ambarella/dsp_print rw,
-/proc/ambarella/hdmi_edid rw,
-/proc/ambarella/cma rw,
+/proc/ambarella/hdmi_edid r,
+/proc/ambarella/cma r,
```
Comment (bboozzoo) [inline interfaces/builtin/dsp.go:70]: A bunch of those was `rw` before, is the snap consuming this interface capable of working properly if a newer snapd lands and updates the interface or do they ensure that the right versions are in place before snapd is updated?

**Review by bboozzoo — APPROVED** [review]

**Review by pedronis — COMMENTED** [review]
this is fine, but would also like confirmation about the the "rw" -> "r" switches?

**Review by anonymouse64 — APPROVED** [review]
lgtm if this is confirmed to be okay with their software

**Comment by woodrow-shen** [comment]
> lgtm if this is confirmed to be okay with their software

My adjustment is based on real system so it should be ok, and they're replying the same result I expressed in the PR.:)

**Pull request merged**